pages/common/csvformat.md

```diff
@@ -0,0 +1,24 @@
+# csvformat
+
+> Convert a CSV file to a custom output format.
+> Included in csvkit.
+
+- Convert to a tab-delimited file:
+
+`csvformat -T {{data.csv}}`
+
+- Convert delimiters to a custom character:
+
+`csvformat -D "{{custom_character}}" {{data.csv}}`
+
+- Convert line endings to carriage return + line feed:
+
+`csvformat -M "{{\r\n}}" {{data.csv}}`
+
+- Convert to minimal use of quote characters:
```

Comment:
Maybe "Convert to minimal use" --> "Minimize use", and append "(omit them wherever unnecessary)" or something else that explains what it does?